### DISCUSSION TITLE
Windows: using another encoder as backend

### DISCUSSION THREAD

**Discussion by xchray** [yermak/AudioBookConverter#628] [Ideas]
I am Windows user. I am recently very happy to to know ABC and happily using it. Before knowing ABC, I am using [qaac ](https://github.com/nu774/qaac/) on windows. Qaac as a pure encoder, it is pretty good since again the backend is Apple QT. Is that Ok to consider a feature to enable external converter to work as backend like foobar2000 does?

Thank you and ABC is awesome.

**Comment by yermak** [comment]
I did consider this at early days... but here is summary from quick googling re original QAAC
To use original QAAC, you need to install iTunes and copy the correct DLLs to the QAAC folder. QAAC requires a number of Apple DLLs to work, but licensing issues prevent these files from being redistributed with QAAC.
GitHub version is not clear for me:


> All files are distributed as Public Domain, except for the followings:
> 
> Files under ALAC, include/ALAC*.h:
> These are part of Apple Lossless Audio Codec Project distribution
> (http://alac.macosforge.org/). Slightly modified.


this require research to check if this could be implemented. As according to qaac licence - ALAC is not in public doamin. 
However, It seems they became opensource at some point - moved under Apache licence...
Also I would need to benchmark performance - assuming quality is better, but need to find balance between quality and speed.

Licencing issues were the main blockers in the past...

**Reply by xchray** [reply]
Thank you for clarification. One last question, if only an interface is defined, user choose any encoder, like mp3 or any, would that still an concern? I guess foobar2k is doing that.
